share/p2p/peers/manager.go

```diff
@@ -24,16 +24,16 @@ import (
 )
 
 const (
-	// ResultSuccess indicates operation was successful and no extra action is required
-	ResultSuccess result = iota
+	// ResultNoop indicates operation was successful and no extra action is required
+	ResultNoop result = "result_noop"
```

Comment:
It was exported before and exported now. Naming changed because it is unclear from caller perspective  how ResultSuccess is different from ResultSynced.